modules/affile/logproto-file-writer.h

```diff
@@ -25,8 +25,9 @@
 #define LOG_PROTO_FILE_WRITER_H_INCLUDED
 
 #include "logproto/logproto-client.h"
+#include "modules/file-rotation/file-rotation.h"
```

Comment:
We should not include file-rotation in the affile module.

`#include "signal-slot-connector/signal-slot-connector.h"` should be enough here.

Reply:
:+1: